extensions/ql-vscode/src/view/variant-analysis/VariantAnalysisHeader.tsx

```diff
@@ -3,49 +3,86 @@ import styled from 'styled-components';
 import { VariantAnalysisStatus } from '../../remote-queries/shared/variant-analysis';
 import { QueryDetails } from './QueryDetails';
 import { VariantAnalysisActions } from './VariantAnalysisActions';
+import { VariantAnalysisStats } from './VariantAnalysisStats';
 
 export type VariantAnalysisHeaderProps = {
   queryName: string;
   queryFileName: string;
   variantAnalysisStatus: VariantAnalysisStatus;
 
+  totalRepositoryCount: number;
```

Comment:
If the user doesn't have access to any of the repositories of the owner, wouldn't it just show the `totalRepositoryCount` as `0`? In this case, it might even do that automatically because the variant analysis will fail which is hardcoded as showing 0.